taxonomies/labels.txt

```diff
@@ -16508,6 +16508,19 @@ xx:NZ Vegetarian Society, NZVS
 en:NZ Vegetarian Society Vegan Certification, NZVS Vegan Certification
 xx:NZ Vegetarian Society Vegan Certification, NZVS Vegan Certification
 
+da:dansk vegetarisk forening
```

Comment:
shouldn't it be under <en:vegetarian too?

Reply:
Yeah, I guess I could add that